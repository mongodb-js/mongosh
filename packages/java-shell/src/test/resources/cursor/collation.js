// before
db.coll.remove({});
db.coll.insertOne({a: "a"});
db.coll.insertOne({a: "A"});
db.coll.insertOne({a: "รก"});
// command
db.coll.find({a: "a"}).collation({"locale": "en_US", strength: 1});
