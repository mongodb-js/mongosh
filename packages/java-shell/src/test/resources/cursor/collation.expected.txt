[ { "_id": <ObjectID>, "a": "a" }, { "_id": <ObjectID>, "a": "A" }, { "_id": <ObjectID>, "a": "\u00E1" } ]